go.mod

```diff
@@ -217,6 +218,8 @@ require (
 	modernc.org/sqlite v1.24.0
 )
 
+require github.com/IBM/nzgo v11.1.0+incompatible // indirect
```

Comment:
Can you please move this down into the other indirect section.

Reply:
Done